database/seed.sql

```diff
@@ -0,0 +1,54 @@
+-- ---
+-- Seed Database
+-- You will need to enable MYSQL variable LOCAL_INFILE in order for these commands to work.**
+-- SET GLOBAL local_infile = 'ON';
+-- **Be sure to list absolute path
+-- Optionally, you can place the files in a specific directory if you wish to load the data without the local flag.  You must edit my.cnf to sidestep any
+-- ---
+LOAD DATA LOCAL INFILE '/Users/malloryburke/hack/rfp2310/ganymede/Products-Service/database/data/product.csv'
+INTO TABLE products
+FIELDS TERMINATED BY ','
+OPTIONALLY ENCLOSED BY '"'
+LINES TERMINATED BY '\n'
+IGNORE 1 LINES; -- Ignore headers
+(item_name, slogan, item_description, category, default_price);
+
+LOAD DATA LOCAL INFILE '/Users/malloryburke/hack/rfp2310/ganymede/Products-Service/database/data/features.csv'
+INTO TABLE features
+FIELDS TERMINATED BY ','
+OPTIONALLY ENCLOSED BY '"'
+LINES TERMINATED BY '\n'
+IGNORE 1 LINES; -- Ignore headers
+(productId,feature,item_value);
```

Comment:
Your schema has a field product_id for features but here the column header looks to be productId. Are these rows being loaded into the database as you would expect?
You might need a line like 
```
SET product_id = productId
```
after line 22.